tests/gold_tests/h2/replay_rst_stream/http2_rst_stream_client_after_data.yaml

```diff
@@ -52,3 +52,21 @@ sessions:
         encoding: plain
         data: server_test
         verify: {as: equal}
+
+  # Unrelated request just to keep the connection open
+  - client-request:
```

Comment:
Looks good. Going to wait for the tests to be changed to utilize the new PV feature that keeps connection open.